packages/kad-dht/src/dual-kad-dht.ts

```diff
@@ -103,7 +103,7 @@ function multiaddrIsPublic (multiaddr: Multiaddr): boolean {
 
   // dns4 or dns6 or dnsaddr
   if (tuples[0][0] === DNS4_CODE || tuples[0][0] === DNS6_CODE || tuples[0][0] === DNSADDR_CODE) {
-    log('%m is public %s', multiaddr, true)
+    log('%a is public %s', multiaddr, true)
```

Comment:
Do we need these messages for general debugging or should they be trace logs?

Reply:
They can probably be removed